distribution/docs/pom.xml

```diff
@@ -33,16 +33,19 @@
             <groupId>org.asciidoctor</groupId>
             <artifactId>asciidoctorj</artifactId>
             <version>${asciidoctorj.version}</version>
+            <scope>provided</scope>
```

Comment:
❔ I assume making the docs `provided` scope means they won't be evaluated by dependency-check?

Reply:
The `compile` scope is transitive and since the distribution has a dependency on the docs, it had a transitive dependency on asciidoctor and jruby as well. This is to prevent that. The `provided` scope isn't transitive, so they no longer leak into projects that depend on the docs.

dependency-check would still scan them (it scans `compile`, `provided`, and `runtime` dependencies), but we skip the docs module.

Reply:
gotcha. Thanks!